api/src/test/java/uk/gov/dhsc/htbhf/claimant/model/VerificationResultTest.java

```diff
@@ -37,6 +37,12 @@ void shouldHaveAddressMismatchForFullAddressMismatch() {
         assertThat(result.isAddressMismatch()).isTrue();
     }
 
+    @Test
+    void shouldNotHaveAddressMismatchWhenNotEligible() {
```

Comment:
This test should be removed too as the VerificationResult should not be able to get in this state.